### ISSUE TITLE
White screen with OpenGL

### ISSUE THREAD

**Issue by ncyglhojimm** [visualboyadvance-m/visualboyadvance-m#428]
### Expected behavior
Play the game with OpenGL.
### Actual behavior
White screen when selecting OpenGL in the GBA options. 
Audio works, controller/keyboard input works.
All Stock settings, so no filters, 100% speed.
VBA-M 2.1.3
### Steps to reproduce the behavior
Change from Simple to OpenGL in GBA settings. Simple works, but with an unplayable stuttering. 

### ROM details
Both GBA:
Pokemon Emerald 
Zelda - Minish Cap

Include screenshots of ROM Information (File > ROM Information):
Files DL'ed from Vimm, should be good ones.

### Option details

Video driver (simple, opengl):
OpenGL
Sound driver (directsound, xaudio, openal, SDL):
SDL

### Build details

Win 10
Vega 64
Radeon 19.4.1 Apr. driver Latest

**Comment by ZachBacon** [comment]
Does it render with the simple driver?

**Comment by rkitover** [comment]
Well, there is no way for us to reproduce something like this, our opengl code is quite simple and works with anything where opengl works. I suggest trying some other programs that use opengl. The simple driver should also not have stuttering on windows. You may be having hardware issues.

**Comment by ncyglhojimm** [comment]
Found the issue with the white screen with OpenGL, it was RivaTuner StatisticsServer. 
I still have issues with constant stuttering with the latest release. I looked around for other builds and Somehow this version https://sourceforge.net/projects/vbam/files/Win32-Wx/VisualBoyAdvance-M-WX-2.0.0-beta2.7z gave me much better performance but I still experience stuttering even though it says it runs at 60 FPS.
